Makefile

```diff
@@ -1,10 +1,14 @@
+SCHEMAS_VERSION=`cat .schemas-version`
 DESIGN_SYSTEM_VERSION=`cat .design-system-version`
 
 clean:
 	rm -rf schemas
 	rm -rf templates/components
 	rm -rf templates/layout
 
+load-schemas:
```

Comment:
I think this needs documenting in `README.md` in `### Setup` similar to how `load-design-system-templates` is documented.